src/configuration/path-alias-utils.ts

```diff
@@ -0,0 +1,46 @@
+import { sync as glob } from "glob";
+import Path from "path";
+import { environment } from "../common/execution-env";
+
+export class PathUtils {
+  public static async getContents <T> (arrayOrPath : T[] | string, parentPath = "") : Promise<T[]> {
+    if(Array.isArray(arrayOrPath)) {
+      const contents = [];
+      for(const entry of arrayOrPath) {
+        contents.push(...(await this.getContent(entry, parentPath)));
+      }
+      return contents;
+    } else if(typeof arrayOrPath === "string") {
```

Comment:
we don't need this second `if` here, neither the `else`.

Reply:
True, I'll change this right away.